### ISSUE TITLE
Arc second conversions

### ISSUE THREAD

**Issue by yegg** [duckduckgo/zeroclickinfo-goodies#826]
e.g. 1 arc second in degrees

https://duck.co/ia/view/conversions

**Comment by venky18** [comment]
```
'convert 1 arc second to degree' => test_zci(
        '1 arc second = 0.000277777777778 degree',
        structured_answer => {
            input     => ['1 arc second'],
            operation => 'convert',
            result    => '0.000277777777778 degree'
        }
    ),
```
what else is needed?

**Comment by mintsoft** [comment]
Nothing, I forgot to close this issue when it was solved in: #1996

**Comment by mintsoft** [comment]
Thanks for the heads up @venky18
